src/components/Book.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+
+function Book({ title, author }) {
+  return (
+    <>
+      <div>
+        <h2>{title}</h2>
+        <h4>{author}</h4>
+      </div>
+    </>
+  );
+}
+Book.propTypes = {
+  title: PropTypes.string,
+  author: PropTypes.string,
+};
+Book.defaultProps = {
+  title: '',
+  author: '',
+};
+
+export default Book;
```

Comment:
Great job so far. Kindly add the `remove`  button to your code since it is needed to remove an individual book from the page and it is one of the project requirements. Kindly, add it.
![image](https://github.com/freddykaberuka/book-store/assets/101523582/aa6099eb-d50f-44c1-b7b3-482519418a7e)